simplq/src/styles/buttons.module.scss

```diff
@@ -30,7 +35,13 @@
   border: 1px solid $primary-color-dark;
 
   &:hover {
-    background-color: #f8f8f8;
+    background: #f8f8f8 radial-gradient(circle, transparent 1%, #f8f8f8 1%) center/15000%;
+  }
+
+  &:active {
+    background-color: #9e99fc;
+    background-size: 100%;
+    transition: background 0s;
```

Comment:
Can you merge upstream new-frontend to this branch, so that these extra diffs go away?